source/localizable/smart_cart/_order_object.html.md.erb

```diff
@@ -94,3 +107,27 @@ Name                                                       | Type    | Descripti
 `line_item_rejection_reason.id`                            | Integer | The rejection reason ID to use
 `line_item_rejection_reason.label`                         | String  | A human readable label
 `line_item_rejection_reason.requires_available_quantity`   | Boolean | Whether or not `available_quantity` is required
+
+#### Invoice details
+
+This attribute is present only when a user requests an invoice for the order. A user
```

Comment:
I think we may remove this: `A user may also request vat exclusion for several line items.`